includes/class-wc-siftscience-admin.php

```diff
@@ -207,7 +207,12 @@ private function get_settings() {
 					'User & Order Name Prefix',
 					'Prefix to give order and user names. '
 					. 'Useful when you have have multiple stores and one Sift account.' ),
+//the added field
+				$this->get_number_input( WC_SiftScience_Options::$min_order_value,
+					'Order Minimum value to Send',
+					 'Orders less than this value wont get sent unless the value is equal to 0', 0 ),
```

Comment:
Change to:
Orders less than this value will not be automatically sent to sift. Set to zero to send all orders.